### ISSUE TITLE
Minimize / Maximize Support

### ISSUE THREAD

**Issue by awcoding** [zzarcon/react-cristal#24]
Would it be possible to get the classic minimize / maximize buttons supported in the header?

**Comment by zzarcon** [comment]
Yeah, I think that's something we can include in the library since we want to be opinionated about the UI

**Comment by awcoding** [comment]
I added the icons w/out the plumbing. I made a PR but looks like github swallowed it. Seems to be some connectivity issues on here today. Here's the tree https://github.com/awcoding/react-cristal
